ant/src/org/jetbrains/kotlin/ant/Kotlin2JvmTask.kt

```diff
@@ -71,6 +71,6 @@ class Kotlin2JvmTask : KotlinCompilerBaseTask() {
 
         if (noStdlib) args.add("-no-stdlib")
         if (noReflect) args.add("-no-reflect")
-        if (includeRuntime) args.add("-include-runtime")
+        if (includeRuntime && output?.extension == "jar") args.add("-include-runtime")
```

Comment:
Could you explain this change?

I think that in the case of an incorrect config, such as `includeRuntime` enabled + directory output, it's better to still pass everything to the compiler as specified by the user, in order for the compiler to handle it in an expected way, such as for example report a warning, than to silently change the compiler arguments here.

With that said, the compiler right now does _not_ report any warning if `-include-runtime` is passed with a directory output, but that is a separate issue (it probably should).

Reply:
I thought that it doesn't make sense passing ```-include-runtime``` to compiler when destination is a jar, obviously I was wrong. It seems the only thing we need to do is to change the default value of  ```includeRuntime``` from ```true``` to ```false```.